unittests/test_schedulers.py

```diff
@@ -746,7 +732,7 @@ def test_flex_alloc_sched_access_constraint_partition(make_flexible_job):
 
 
 def test_flex_alloc_sched_access_partition(make_flexible_job):
-    job = make_flexible_job('all', sched_access=['--partition=p1'])
+    job = make_flexible_job('all')
```

Comment:
Should you pass an extra job option here, like you do for the `test_flex_alloc_partition_idle` unittest?

Reply:
@rsarm This should not change. Check my comment above.